news/2024/2024-07-23-osu-austria-conventions-2024.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: osu! Austria @ Conventions 2024
+date: 2024-07-23 12:00:00 +0000
+---
+
+The osu! Austria community proudly presents its appearance at not one, but two upcoming conventions in 2024!
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/Banner.png)
+
+The Austrian osu! community is excited to announce their presence at two upcoming conventions. Next to osu! booths, there will also be LAN tournaments, in which you can show off your skills and win some nice goodies. So what are you waiting for? Get all the information you need down below!
+
+## AniNite 2024
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/aninite.png)
+
+Join us at the biggest Anime and Manga convention in Austria, [AniNite](https://www.aninite.at/en/), which has over 25,000 yearly visitors. Meet our community from **August 9th to 11th, 2024** at our osu! booth. In cooperation with [Friendly Fire](https://friendlyfireesports.com/), we have organized 12 PCs for you to play on and show everyone what you're made of.
+
+During this time, a **battle royale-styled** osu! tournament will be hosted. Make sure to register during the event and take home the trophy!
+
+## YuniCon 2024
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicon.png)
+
+Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organized computers provided by [Streamhouse](https://streamhouse.at/)!
+
+On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament Austria has ever seen! The top players of the tournament will ferociously duke it out for some handsome physical prizes and a bespoke profile badge on the main stage of the event.
+Top 16 players won't go home empty-handed as they will receive some nice goodies as well!
```

Comment:
this doesnt have to be a new sentence, it could be tacked onto the existing one

Reply:
did something here